src/coreclr/nativeaot/BuildIntegration/BuildIntegration.proj

```diff
@@ -42,10 +42,7 @@
     </ItemGroup>
     <Copy SourceFiles="@(AnalyzerFile)" DestinationFolder="$(RuntimeBinDir)\analyzers\dotnet\cs" />
   </Target>
-<<<<<<< HEAD
-=======
 
-  <Target Name="Restore" />
+  <Target Name="Restore" Condition="'$(TargetsWasm)' != 'true'"/>
```

Comment:
Does this work correctly? I though MSBuild would override this target anyway, it just wouldn't 'run' it under the condition (which for this no-op target would not be meaningful).

I think we should just remove this in our fork; it's needed upstream because the upstream project doesn't use the traversal SDK.

Reply:
Hmm, not sure to be honest.  Have removed it.